Form-Controls/styles.css

```diff
@@ -0,0 +1,47 @@
+
+h1{
+    text-align: center;
+    color: #354f52;
+}
+
+h2{ 
+    text-align: center;
+    color: #354f52;
+    font-size: 12px;
```

Comment:
(Don't use font sizes less than 16px for most things.)

Are pixel sizes the best choice for fonts? I think this lesson would be good for you to explore

https://web.dev/learn/design/typography/